pkg/kubelet/types/pod_update.go

```diff
@@ -142,18 +138,13 @@ func (sp SyncPodType) String() string {
 	}
 }
 
-// IsCriticalPod returns true if the pod bears the critical pod annotation key or if pod's priority is greater than
-// or equal to SystemCriticalPriority. Both the default scheduler and the kubelet use this function
-// to make admission and scheduling decisions.
+// IsCriticalPod returns true if pod's priority is greater than or equal to SystemCriticalPriority.
+// Both the default scheduler and the kubelet use this function to make admission and scheduling
```

Comment:
Please delete lines 142 and 143. The scheduler does not really care about criticality of pods. From the scheduler's point of view a pod with a higher priority is more important than pods with lower priority.